### DISCUSSION TITLE
How is it accessing the internet ?

### DISCUSSION THREAD

**Discussion by arian0zen** [reworkd/AgentGPT#278] [Q&A]
I tried it out and it can pretty much do web scraping smoothly, I was wondering how is it doing that?
and in which file that code is written ?

**Comment by Besix2** [comment]
Its GPT-4 it has the ability to do that.

**Reply by stefangrotz** [reply]
Only GPT-4 plus the new Plugin beta functionality could do that and i doubt that this is used here. But it also works using GPT-3.5, so there must be something else.
